tests/resources/Util.robot

```diff
@@ -150,6 +173,12 @@ Text Input
     Wait Until Element Is Visible And Enabled  ${element_xpath}
     Input Text  ${element_xpath}  ${text}
 
+Clear Field Of Characters
```

Comment:
can we leverage the buildin method? https://github.com/goharbor/harbor/blob/master/tests/robot-indices/Selenium2Library.index#L11

Reply:
It does not work as expected, so I left old code in use.